### DISCUSSION TITLE
Screen flickering when switching refresh rates

### DISCUSSION THREAD

**Discussion by nishantpajni** [tribalfs/GalaxyMaxHzPub#245] [Q&A]
I have an A34 and there are only two options to choose from, 60 and 120. When the adaptive refresh rate drops to 60 Hz, I can see that the screen is darker than when it's at 120 Hz. It's very noticeable and the constant flickering b/w light and dark is distracting. Kindly address this issue as with this, it is nigh unusable for someone like me.

**Comment by tribalfs** [comment]
https://github.com/tribalfs/GalaxyMaxHzPub/wiki/Frequently-Asked-Questions-(FAQs)#8-why-sometimes-i-notice-my-color-shift-when-refresh-rate-switches-in-low-brightness-whats-the-workaround
